acct-group/spamd/metadata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="person">
+    <email>gentoo_bugs_peep@parallaxshift.com</email>
+    <name>Philippe Chaintreuil</name>
+  </maintainer>
+
+  <maintainer type="person">
+    <email>bug@mejor.pl</email>
+    <name>Marcin Mirosław</name>
+  </maintainer>
```

Comment:
Who is this? Do you have their consent on being added as a maintainer?

Reply:
Marcin is co-proxy maintainer of spamassassin:
 https://bugs.gentoo.org/583908

But I have not gotten his explicit consent for the user/group packages.